src/Tablebot/Plugins/Flip.hs

```diff
@@ -0,0 +1,43 @@
+{-|
+Module      : Tablebot.Plugins.Ping
```

Comment:
Very minor thing; both Flip.hs and Random.hs have incorrect names for the module in the documentation

Reply:
As is tradition.